generation/new_client/README.md

```diff
@@ -172,17 +172,25 @@ Example: `https://cloud.google.com/alloydb/docs`
 
 ### REST Docs
 
-The corresponding value in the Cloud Drop page is `documentation_uri`.
+This link must be found manually (until b/310975675 is implemented), and may not exist for every library, as they are generated adhoc.
```

Comment:
Avoid starting a paragraph with "Note that..." sentence.

Reply:
Updated as there's now an official Cloud Drop field for it